.github/workflows/docker.yml

```diff
@@ -39,11 +39,15 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
+      - name: Get short commit SHA1
+        shell: bash
+        run: |
+          echo "sha_short=$(git rev-parse --short "$GITHUB_SHA")" >> "$GITHUB_ENV"
```

Comment:
`GITHUB_SHA` looks weird.

Reply:
According to the [docs](https://docs.github.com/en/actions/learn-github-actions/variables)
<img width="701" alt="Screenshot 2024-01-05 at 2 46 17 AM" src="https://github.com/sysprog21/rv32emu/assets/9054879/ed966096-3455-4eae-b971-d2f4904fc9ee">

This returns the full commit hash. 

Is there anything weird with it?

Reply:
It would be short hash instead of short sha.
https://git-scm.com/docs/git-rev-parse

Reply:
Sorry, I am not sure if I understand this correctly. 

Isn't the commit SHA, which is the hash that identifies a specific commit, after passing it through `git rev-parse --short`, a short version of the commit SHA (a.k.a the short hash) will be returned?

As the GitHub CI environmental variable doesn't provide a short commit SHA, and the full string is named `GITHUB_SHA` by them, I can't change the name of it.

Reply:
> Isn't the commit SHA, which is the hash that identifies a specific commit, after passing it through `git rev-parse --short`, a short version of the commit SHA (a.k.a the short hash) will be returned?

I was concerned about the naming scheme instead of the logic behind this pull request.

Reply:
Updated